types/src/transaction/multisig.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Aptos
```

Comment:
Sad, I wish we didn't have the duplication of types here and in `api/`. This is fine, we'll fix this up eventually...

Reply:
Yeah this is an unfortunate duplication that stems from EntryFunction I think. This is not easy to fix as it'd be a backward-incompatible change. So we might need to live with this forever...